src/content/docs/new-relic-solutions/business-observability/intro-pathpoint.mdx

```diff
@@ -192,3 +201,242 @@ Beyond structured signals tied to New Relic entities or alerts, Pathpoint suppor
+
+It’s better to have a well thought out flow with some signal gaps than a flow that is put together hastily with a random set of signals.
+
+**Feel free to use multiple flows if your Journey is very complex**
+
+Trying to force all relevant signals into a single flow can lead to a congested and confusing signal.  Consult with internal stakeholders (and the NR account team) to see how best to organize flows.  For example* an insurance company way want to might separate the following flows:
+
+- Finding and purchasing new policies
+- Filing and paying out insurance claims
+
+**Don’t use a signal just because it’s available**
+
+Be selective and focus on the signals that are most likely related to immediate business or user impact.   You may have 5 alert conditions related to an entity, however you may choose to only add 1 or 2. 
+
+**Don’t be afraid to improve existing signals**
+
+Flapping signals will create a noisy, flapping Pathpoint flow.  Consider refining alert thresholds or service level queries to make signals more actionable and meaningful.  If a signal is red, there should be real or at least probably risk to the business process.  
+
+**Use Synthetics to capture external dependencies**
+
+Your process may be responsible for 3 or 4 third party APIs or other SaaS systems.  Synthetics is the conventional way to capture the health of those systems. 
+
+**Use Workloads to group entities**
+
+Workloads are a powerful means to create aggregated entities where if one or several sub-entities have critical issues we want that overall workload to show as critical.  This can be done to capture large pools of related infrastructure (things like load balancers, RDBMs, other cloud services).    Another tip is to create workloads that group multiple service levels into a single aggregate service level.
+
+**Use useful metadata tags on your entities**
+
+It is good practice to put tags on your entities such as service levels, workloads, synthetics etc.   It is especially helpful to add flow, stage, and step tags on your entities.  This will allow you to use these signals more easily with Pathpoint.
+
+**Use the Pathpoint KPIs row to capture important counts, $ amounts, etc. related to your flow**
+
+Some organizations want to have counts for important critical events like user logins, user registrations, successful checkouts, carts abandoned as well as important dollar amounts like total checkout value, total returned product value.   You can usually find some of these in your existing telemetry and they don’t have to be associated with an entity.  They can be logs, custom metrics, etc.
+
+Let’s say you have a stage called “Search” you may want to have KPI’s like:
+
+- Successful Searchs
+- Slow Searches
+- Failed Searches
+
+If you have stages for Order, Cart, or Checkout you may want to have KPIs like:
+
+- Order Count
+- Failed Orders
+- Order Value
+- Abandoned Cart
+- Average Cart Value
```

Comment:
```suggestion
**Work from the top down**

When designing our flow, the reason we start from the top down is we may not have all of the signals needed yet to properly cover the flow. During flow development, we'll identify areas were we do and do not have coverage.

It’s better to have a well thought out flow with some signal gaps than a flow that's put together hastily with a random set of signals.

**Use multiple flows if your journey is very complex**

Trying to force all relevant signals into a single flow can lead to a congested and confusing signal. Consult with internal stakeholders (and the New Relic account team) to see how best to organize flows. For example, an insurance company might want to separate the following flows:

- Finding and purchasing new policies
- Filing and paying out insurance claims

**Don’t use a signal just because it’s available**

Be selective and focus on the signals that are most likely related to immediate business or user impact.  You may have 5 alert conditions related to an entity, however you may choose to only add 1 or 2. 

**Don’t be afraid to improve existing signals**

Flapping signals will create a noisy, flapping Pathpoint flow. Consider refining alert thresholds or service level queries to make signals more actionable and meaningful. If a signal is red, there should be real or at least probably risk to the business process. 

**Use Synthetics to capture external dependencies**

Your process may be responsible for 3 or 4 third-party APIs or other SaaS systems. Synthetics is the conventional way to capture the health of those systems.

**Use Workloads to group entities**

Workloads offer a powerful method to consolidate related entities into a unified view. By aggregating infrastructure components such as load balancers, RDBMS, and other cloud services, critical issues affecting individual sub-entities can be efficiently surfaced at the workload level. Additionally, workloads can be used to combine multiple service levels into a single, overarching service level for simplified monitoring and management.

**Use useful metadata tags on your entities**

It's good practice to put tags on your entities such as service levels, workloads, synthetics etc. It's especially helpful to add flow, stage, and step tags on your entities. This will allow you to use these signals more easily with Pathpoint.

**Use the Pathpoint KPIs row to capture important counts, monetary values, etc. related to your flow**

Utilize the Pathpoint KPIs row to monitor essential counts and monetary values relevant to your workflow. This includes critical events such as user logins, registrations, checkouts, and cart abandonments, as well as financial metrics like total checkout and returned product value. Many of these metrics can be derived from existing telemetry data and do not require association with specific entities. They can be sourced from logs, custom metrics, or other available data points.

Let’s say you have a stage called “Search”, you may want to have KPI’s like:

- Successful searches
- Slow searches
- Failed searches

If you have stages for order, cart, or checkout, you may want to have KPIs like:

- Order count
- Failed orders
- Order value
- Abandoned cart
- Average cart value
```